### PR TITLE
gui basic

### PR DESCRIPTION
**Pull request by edussuriyac**
# PR for issue #20

### CODE REVIEW

File: src/main/resources/scenes/NewDownload.fxml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (kasvith) [inline src/main/resources/scenes/NewDownload.fxml]: name should be simple like `newDownloadDialog.fxml`

File: src/main/resources/scenes/MainController.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package sample;
+
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.stage.Stage;
+
+public class MainController {
+
+    public void newDownload(){
```
Comment (kasvith) [inline src/main/resources/scenes/MainController.java]:
```suggestion
    public void newDownloadDialog(){
```

File: src/main/resources/scenes/octopus.fxml [hunk 3]
```diff
@@ -1,32 +1,127 @@
...
+                     </HBox.margin>
+                  </Button>
+                  <Button mnemonicParsing="false" prefHeight="24.0" prefWidth="24.0">
+                     <HBox.margin>
+                        <Insets right="8.0" />
+                     </HBox.margin>
+                     <graphic>
+                        <ImageView fitHeight="15.0" fitWidth="16.0" pickOnBounds="true" preserveRatio="true">
+                           <image>
+                              <Image url="@../../octopusdownloader-master/src/main/resources/images/kisspng-computer-icons-youtube-play-button-clip-art-icon-png-play-button-5ab1113388a855.8771330715215537155598.jpg" />
```
Comment (kasvith) [inline src/main/resources/scenes/octopus.fxml]: can you rename images to simple ones like `play.png`, `pause.png` etc

**Comment by kasvith** [comment]
also add some screenshot

**Comment by kasvith** [comment]
update your PR with latest changes

File: src/main/resources/scenes/MainController.java (hunk 2 above)
Comment (kasvith) [reply src/main/resources/scenes/MainController.java]: Can you fix these too

**Comment by edussuriyac** [comment]
okayzz

**Comment by kasvith** [comment]
tnx @edussuriyac

**Pull request merged**